src/lib/util/feature-evaluator/constraint.ts

```diff
@@ -39,10 +39,15 @@ const cleanValues = (values: string[]) =>
 
 const InOperator = (constraint: Constraint, context: Context) => {
     const field = constraint.contextName;
+    const caseInsensitive = Boolean(constraint.caseInsensitive);
     const values = cleanValues(constraint.values);
     const contextValue = resolveContextValue(context, field);
 
-    const isIn = values.some((val) => val === contextValue);
+    const isIn = values.some((val) =>
+        caseInsensitive
+            ? val.toLowerCase() === contextValue?.toLowerCase()
```

Comment:
IIRC we don't have any we have any restrictions on unicode so localCompare or toLocaleLowerCase might be better here